packages/cli/src/zosjobs/modify/job/Job.handler.ts

```diff
@@ -0,0 +1,67 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { IHandlerParameters, ImperativeExpect } from "@zowe/imperative";
+import { ModifyJobs, GetJobs, IJob } from "@zowe/zos-jobs-for-zowe-sdk";
+import { ZosmfBaseHandler } from "@zowe/zosmf-for-zowe-sdk";
+
+/**
+ * "zos-jobs modify job" command handler. Modify a job by name and ID.
+ * @export
+ * @class JobHandler
+ * @implements {ICommandHandler}
+ */
+export default class JobHandler extends ZosmfBaseHandler {
+    /**
+     * Command line arguments passed
+     * @private
+     * @type {*}
+     * @memberof JobHandler
+     */
+    private arguments: any;
+
+    /**
+     * Command handler process - invoked by the command processor to handle the "zos-jobs modify job"
+     * @param {IHandlerParameters} params - Command handler parameters
+     * @returns {Promise<void>} - Fulfilled when the command completes successfully OR rejected with imperative error
+     * @memberof JobHandler
+     */
+    public async processCmd(params: IHandlerParameters): Promise<void> {
+        this.arguments = params.arguments;
+
+        // Force yargs parameters to be proper types
+        const jobid: string = this.arguments.jobid + "";
```

Comment:
Just a suggestion, but you could just append `.toString()` to verify this is a string in most cases if you are trying to force them to be the proper type. This doesn't need to be implemented since it is documented what the `+ ""` is doing. 😃